src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java

```diff
@@ -178,7 +193,9 @@ public void postEholdingsPackages(String contentType, PackagePostRequest entity,
       .requestAction(context ->
         getVendorId(context)
           .thenCompose(id -> context.getPackagesService().postPackage(packagePost, id))
-          .thenCompose(packageById -> updateTags(packageById, context.getOkapiData().getTenant(), tags))
+          .thenCompose(packageById ->
```

Comment:
@andrii-paias : Confused as to why we are dealing with tags on POST package endpoint at all? When I try to create a new package(which is the only time a POST is being invoked), the only fields that I see are name, content type and coverage settings. I don't think we have to do anything with tags at all in the POST request unless I m missing something.

Reply:
It's probably because initially it wasn't obvious in which situations tags can be updated, so POST/PUT requests have tags attribute, now that we know that tags are always updated independently from packages, we can probably remove tag update from POST and PUT,
I added a story for this - https://issues.folio.org/browse/MODKBEKBJ-283

Reply:
perfect, thanks 👍